back/spec/acceptance/phase_insights/common_ground_phase_insights_spec.rb

```diff
@@ -28,12 +28,15 @@
     )
   end
 
+  # Fixing the dates used as relative to this reference time means we can expect exact dates in the chart data
+  let(:time_now) { Time.new(2025, 12, 2, 12, 0, 0) }
```

Comment:
You could consider wrapping the whole test in: `travel_to(Date.parse('2025-12-12')) do` instead and keeping `3.days.ago` etc? We use this across quite a few tests